src/core/colorspace.js

```diff
@@ -1046,24 +1097,24 @@ const CalRGBCS = (function CalRGBCSClosure() {
     if (color >= 0.99554525) {
       return 1;
     }
-    return adjustToRange(0, 1, (1 + 0.055) * color ** (1 / 2.4) - 0.055);
+    return this.#adjustToRange(0, 1, (1 + 0.055) * color ** (1 / 2.4) - 0.055);
   }
 
-  function adjustToRange(min, max, value) {
+  #adjustToRange(min, max, value) {
     return Math.max(min, Math.min(max, value));
   }
 
-  function decodeL(L) {
+  #decodeL(L) {
     if (L < 0) {
-      return -decodeL(-L);
+      return -this.decodeL(-L);
```

Comment:
Shouldn't this be `this.#decodeL` (note the `#`) here?

Reply:
D'oh, of course it should; thanks for spotting this!

This oversight unfortunately suggests that we've got no reference tests where `L < 0`.
Also, this shows why you want to use private methods/fields as much as possible since ESLint is actually able to flag unused or misspelled private ones (but cannot do so for "normal" ones since it's a static code analyzer).